testing/integration/integration_util.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/google/exposure-notifications-server/internal/database"
+	"github.com/google/exposure-notifications-server/internal/monolith"
+)
+
+func SetEnvAndRunServer(tb testing.TB, ctx context.Context, dbConfig *database.Config) {
+
+	// Set all of these to not need to connect to external resources.
```

Comment:
FYI (no changes needed), but https://github.com/google/exposure-notifications-server/pull/549 will make this easier. You can construct the config object instead of setting the envvars.

Reply:
Exciting!